cpp/tests/transform/mask_to_bools_test.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, NVIDIA CORPORATION.
+ * Copyright (c) 2020-2021, NVIDIA CORPORATION.
```

Comment:
Do you have script or hack to update Copyright year in all files?

Reply:
No.